README.md

```diff
@@ -37,8 +37,8 @@ api_secret = <api_secret>
 * Open a new terminal window and move to the *ui* folder and install dependencies - `cd ui && npm install`
 * create a `.env` file in the root of `ui/` and add the following inside it
 
-```.env
-REACT_APP_BASE_URL = http://localhost:8080
+```bash
+REACT_APP_BASE_URL='http://localhost:8080'
```

Comment:
"I faced issues when i put quotes for string literals in the env. Are you sure this works?"

Ignore this if its working fine.